app/src/main/res/drawable/ic_autoplay_disabled.xml

```diff
@@ -8,9 +8,9 @@
     android:viewportWidth="24"
     android:viewportHeight="24">
     <path
-        android:fillColor="@color/disabled_text"
+        android:fillColor="?primaryText"
```

Comment:
Does using a text color reference pose any risk being used for a vector? (e.g. the primary text color is updated by Design, but the disabled icon color stays the same) Or will the two colors always be the same?

Reply:
We are probably on the latter side of things. Most of our icons appear directly next to text which usually follow the same text colour. Of course, it will generally be great to define variables such as `primaryIconFillColor` and refactor everything, but that is probably beyond the scope here.